src/Command/Branch/BranchChangelogCommand.php

```diff
@@ -61,6 +63,55 @@ protected function configure()
 To learn more about composing your own regex patterns see:
 http://php.net/manual/reference.pcre.pattern.syntax.php
 http://www.regular-expressions.info/
+
+Changelog format
+----------------
+
+The default changelog format is <comment>title ([#id](url))</comment>
+which is perfectly usable for markdown. But you can easily change this 
+to any format you want, in fact you can even the entire template.
+
+Add the following to your local <comment>.gush.yml</comment> file:
+
+<comment>
+templates:
+    changelog: |
+        {% for item in items %}
+        * {{ item.title }} ([{{ item.id }}]({{ item.url }})) by {{ item.user }}
+        {% endfor %}
+</comment>
+
+The <comment>templates.changelog</comment> expects a Twig template as
+string, or an array pointing to a local file (relative to your repository
+directory).
+
+<comment>
+templates:
+    changelog: [".gush/templates/changelog.twig"]
```

Comment:
i am not too fond of this one, keep it simple rather with templates.changelog key, it does not need to redirect to a file.